spec/abci/abci++_basic_concepts.md

```diff
@@ -375,17 +374,21 @@ the chain once they are verified by a subset of validators. These evidences will
 passed on to the Application through ABCI. It is the responsibility of the
 Application to handle evidence of misbehavior and exercise punishment.
 
-There are two forms of evidence: Duplicate Vote and Light Client Attack. More
+There are two forms of evidence of misbehavior: Duplicate Vote and Light Client Attack. More
 information can be found in either [data structures](../core/data_structures.md)
 or [accountability](../light-client/accountability/).
```

Comment:
There is a documentation for evidence, maybe it is better to refer to it: https://github.com/cometbft/cometbft/blob/3301699a68f477bfea7e7c048cf955b88e6062b6/spec/consensus/evidence.md

Reply:
This accountability spec is super complex to follow, although pretty comprehensive.

Reply:
good feedback, pointed to the evidence doc in consensus